### PR TITLE
Preview workflow to checkout repo and attempt helm deploy

### PR DESCRIPTION
**Pull request by ellie**
This will fail as I haven't set the auth yet, but let's see if we can at
least fail in the right way :D

## Type of change
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Checklist
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by ellie** [comment]
I very much doubt GitHub Actions will run this without it being merged, but I can try right?

**Comment by ellie** [comment]
@deploy

**Comment by ellie** [comment]
!deploy

File: .github/workflows/pr-deploy.yaml [hunk 1]
```diff
@@ -22,14 +22,34 @@ jobs:
...
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        if: steps.check.outputs.has-permission && ${{ github.event.comment.body == '!deploy'}}
+
+      - name: Declare variables that we can share across steps
+        id: vars
+        run: |
+          GIT_SHORT="$(git rev-parse --short HEAD)"
+          echo "::set-output name=git_short::${GIT_SHORT}"
+          echo "::set-output name=fqdn_record::${GIT_SHORT}.posthog.cc"
+        if: steps.check.outputs.has-permission && ${{ github.event.comment.body == '!deploy'}}
```
Comment (hazzadous) [inline .github/workflows/pr-deploy.yaml:41]: I think this should be the below but I'm not sure.

```suggestion
        if: ${{ steps.check.outputs.has-permission && github.event.comment.body == '!deploy' }}
```

Comment (ellie) [reply .github/workflows/pr-deploy.yaml:41]: I _think_ you might be right, kinda wishing I could test this without pushing it up 🙃

**Comment by ellie** [comment]
!deploy

File: .github/workflows/pr-deploy.yaml (hunk 1 above)
Comment (hazzadous) [reply .github/workflows/pr-deploy.yaml]: Yeah it's not a great dev experience :(

**Pull request merged**